### PR TITLE
Python scripts

### PR DESCRIPTION
**Pull request by peabody124**
Various scripts using the python tools

### CODE REVIEW

File: python/visualize/vid_overlay_position.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        sys.path.insert(1, os.path.dirname(sys.path[0]))
+        from taulabs import telemetry
+        uavo_list = telemetry.GetUavoBasedOnArgs()
+        from taulabs.uavo import UAVO_PositionActual, UAVO_NEDPosition, UAVO_PathDesired
+
+        pa = uavo_list.as_numpy_array(UAVO_PositionActual)
+        ned = uavo_list.as_numpy_array(UAVO_NEDPosition)
+        pos_des = uavo_list.as_numpy_array(UAVO_PathDesired)
+
+        out = "position_overlay.mp4"
```
Comment (peabody124) [inline python/visualize/vid_overlay_position.py]: @mlyle if i were less terrible at this, how would i make this an optional command line argument?

Comment (BrainFPV) [inline python/visualize/vid_overlay_position.py]: Have a look at `argparse`, e.g., in the code by @mlyle below

https://github.com/TauLabs/TauLabs/blob/next/python/taulabs/telemetry.py#L369

Comment (mlyle) [inline python/visualize/vid_overlay_position.py]: Yes, I agree-- steal that code.  (I stole it myself heh)

Comment (BrainFPV) [inline python/visualize/vid_overlay_position.py]: Stealing code is unethical, especially if it is from other open source projects or even more so from other source files within the same project.

/jk

**Comment by mlyle** [comment]
Note I expect a small conflict with #1632 -- no biggie.

**Comment by mlyle** [comment]
Imported into my pull request #1638 to eliminate conflict.

**Comment by mlyle** [comment]
Also adjusted the visualizer to be executable.

**Comment by peabody124** [comment]
Closing in favor of #1638

**Pull request closed without merging**